x-pack/test/functional/apps/security/secure_roles_perm.ts

```diff
@@ -89,6 +91,7 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
     after(async function () {
       // NOTE: Logout needs to happen before anything else to avoid flaky behavior
       await PageObjects.security.forceLogout();
+      await security.testUser.restoreDefaults();
```

Comment:
Same as above:

```diff
diff --git a/x-pack/test/functional/apps/security/secure_roles_perm.ts b/x-pack/test/functional/apps/security/secure_roles_perm.ts
index 05016018e83..429560ee59a 100644
--- a/x-pack/test/functional/apps/security/secure_roles_perm.ts
+++ b/x-pack/test/functional/apps/security/secure_roles_perm.ts
@@ -91,10 +91,10 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
     after(async function () {
       // NOTE: Logout needs to happen before anything else to avoid flaky behavior
       await PageObjects.security.forceLogout();
-      await security.testUser.restoreDefaults();
       await kibanaServer.importExport.unload(
         'x-pack/test/functional/fixtures/kbn_archiver/security/discover'
       );
+      await security.testUser.restoreDefaults();
     });
   });
 }
```

Reply:
Fixed